packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -14,6 +14,7 @@ type error =
   | OperationError(Operation.Error.t)
   | PointSetConversionError(SampleSetDist.pointsetConversionError)
   | SparklineError(PointSetTypes.sparklineError) // This type of error is for when we find a sparkline of a discrete distribution. This should probably at some point be actually implemented
+  | LogarithmOfDistributionError(string)
```

Comment:
It would be nice if errors didn't have to rely on strings in their data types if they didn't need to. Makes it easier to check for specific errors + change the errors in the future if need to. Although this is far off from current features, this would need to change if we wanted to do any form of internationalisation or send errors over a network etc